blink/machine.h

```diff
@@ -161,14 +161,23 @@ struct MachineMemstat {
   memstat_t pagetables;
 };
 
+// Segment descriptor cache
+// @see Intel Manual V3A §3.4.3
+// Currently only the base address for each segment register is maintained;
+// a full implementation will also cache its limit & access rights
+struct DescriptorCache {
+  u16 sel;   // visible selector value (or paragraph value in real mode)
+  u64 base;  // base linear address
```

Comment:
Nit but there's six bytes of wasted space here. You could create two separate arrays.

Reply:
It's not worth thinking that much about.